package.json

```diff
@@ -23,29 +23,25 @@
     "saucelabs": "ng test ng-bootstrap -c saucelabs",
     "scripts:tdd": "ts-node-dev --respawn --project misc/tsconfig.json node_modules/jasmine/bin/jasmine misc/*.spec.ts",
     "scripts:test": "ts-node --project misc/tsconfig.json node_modules/jasmine/bin/jasmine misc/*.spec.ts",
-    "playwright": "node misc/playwright.js",
     "e2e-app:serve": "ng serve e2e-app",
     "e2e-app:lint": "ng lint e2e-app",
     "ngb:static": "ts-node --project misc/tsconfig.json misc/copy-static-files.ts",
     "ngb:lint": "ng lint ng-bootstrap",
     "ngb:test": "ng test ng-bootstrap --code-coverage --source-map true --progress false --watch false",
     "ngb:tdd": "ng test ng-bootstrap --source-map false",
-    "ngb:e2e": "concurrently --success first --kill-others \"yarn e2e-app:serve -c playwright\" \"yarn ngb:e2e-playwright && yarn ngb:e2e-playwright -b firefox && yarn ngb:e2e-playwright -b webkit\"",
-    "ngb:e2e-playwright": "yarn playwright",
+    "ngb:e2e": "playwright test -c e2e-app",
```

Comment:
I think we can remove `concurrently` as well from dependencies

Reply:
Indeed, thank you!